packages/reducers/src/combineReducers.test.js

```diff
@@ -0,0 +1,125 @@
+import { createStore } from 'redux';
+import combineReducers from './combineReducers';
+
+describe('Utils', () => {
+	describe('combineReducers', () => {
```

Comment:
i'm missing a test for the following behaviour:

`it('preserves unknown keys in the state'`

contact me if you don't understand what should be the purpose of this test

Reply:
i think that it will definitely result in some changes in your combineReducers implementation :) i'm curious in how you're going to tackle that!